node-zerox/src/utils.ts

```diff
@@ -104,3 +108,25 @@ export const convertPdfToImages = async ({
     console.error("Error during PDF conversion:", err);
   }
 };
+
+// Convert each page (from docx) to a png and save that image to tmp
+export const convertDocxToPdf = async ({
```

Comment:
seems like the wrong naming convention if `convertDocxToPdf` also works with PPT & Doc.